.github/workflows/deploy_nightly_docs.yml

```diff
@@ -16,13 +16,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Download artifact
-        uses: actions/download-artifact@v4.1.7
+        uses: actions/download-artifact@v4
         with:
             name: "te_docs"
             path: "html"
       - name: Prepare for pages
         uses: actions/upload-pages-artifact@v1.0.7
         with:
+          name: github-pages
```

Comment:
Looking at logs before we changed the artifact action versions, we were experiencing failures due to mismatched artifact names:
https://github.com/NVIDIA/TransformerEngine/actions/runs/10686862322/job/29623182933